discovery-provider/src/challenges/profile_challenge.py

```diff
@@ -0,0 +1,116 @@
+from collections import Counter
+from src.models import ProfileCompletionChallenge, Challenge, User, AggregateUser, Repost, UserChallenge, Follow, Save
+from src.challenges.challenge_event_bus import ChallengeEvent
+from src.challenges.challenge import ChallengeManager, ChallengeUpdater
+
+REPOST_THRESHOLD = 1
+FOLLOW_THRESHOLD = 5
+FAVORITES_THRESHOLD = 1
+
+class ProfileChallengeUpdater(ChallengeUpdater):
```

Comment:
worth a doc-string explaining what the challenge is / how it works?

Reply:
did we add?

Reply:
ty